cos_alerter/alerter.py

```diff
@@ -0,0 +1,53 @@
+import fcntl
+import json
+import tomllib
+
+import apprise
+
+
+class Config:
+    def __getitem__(self, key):
+        with open('/etc/cos-alerter.toml', 'rb') as f:
+            return tomllib.load(f)[key]
+
+
+config = Config()
+
+
+class DataWriter:
```

Comment:
**question:** This is called `Writer` but it does `Reader` things. I'm confused.

Reply:
`InMemoryConfig`?

Reply:
`AlerterState` it is